### ISSUE TITLE
Need to support bcrypt as password storage method

### ISSUE THREAD

**Issue by blandyuk** [YodasMyDad/mvcforum#133]
Hi all,

In order for MVCForum to be as secure as possible, it would be very wise to store ALL passwords on the database using bcrypt. I recommend using $12$ or 1 << 12 (4096) iterations of bcrypt to make it even more horrible for crackers as bcrypt does not optimize on GPUs at all.

**Comment by blandyuk** [comment]
OK, I've updated v1.9.2 locally on my version and its working perfectly. Example of default admin password hash:

$2a$12$ad5FvIJSIO4vudQGK6ssGOY1bPPQxFXQl99Nr5I/Q4o50p6ZWRWjC
Password = password

Using Nuget package "BCrypt" by Fabian Vilers (Dev One) on "MVCForum.Utilities" in "StringUtils" class. I depreciated the old SHA256Managed() salted hash due to it be very insecure. I cannot update github so this is only available to me for now unless someone wants it?
